tests/components/onewire/const.py

```diff
@@ -775,6 +775,36 @@
             },
         ],
     },
+    "7E.222222222222": {
+        "inject_reads": [
+            b"EDS",  # read type
+            b"EDS0066",  # read device_type - note EDS specific
+        ],
+        "device_info": {
+            "identifiers": {(DOMAIN, "7E.222222222222")},
+            "manufacturer": "Maxim Integrated",
+            "model": "EDS",
+            "name": "7E.222222222222",
+        },
+        SENSOR_DOMAIN: [
+            {
+                "entity_id": "sensor.7e_222222222222_temperature",
```

Comment:
is it "7e" or "7E" it is confusing.

I would have asked you to make a constant for "7E.222222222222", since it is repeated multiple times possibility or errors. However I can see you follow the pattern in the file so that change is for later.
```suggestion
                "entity_id": "sensor.7E_222222222222_temperature",
```